tests/unit/configuration_test.py

```diff
@@ -2184,7 +2184,8 @@ def test_deprecated_param_names(self):
                         "workers": 4
                     }
                 ],
-                "default_workers_per_api_key": 2
+                "default_workers_per_api_key": 2,
+                "use_multiprocess_copying_workers": false
```

Comment:
Perhaps also a good idea to add a comment here / similar adding a note that we are explicitly testing this scenario.